packages/mobile/locales/base/translation.json

```diff
@@ -666,6 +666,17 @@
     "step2": "With {{appName}}, it’s easy to access and easy to grow your crypto.",
     "step3": "All you need is a mobile phone to send, spend and explore."
   },
+  "enableBiometry": {
+    "title": "Secure your wallet",
+    "description": "Face ID secures your wallet and makes it easier to use Valora",
```

Comment:
ah you're right, i forgot 🙈 i've updated this key and also modified how we handle the other keys (i opted to translate each biometrics option separately to avoid having multiple variations for each biometrics option on every key we need. unfortunately the biometrics options we get from the keychain lib is not exactly what we need)